test/honest_test.go

```diff
@@ -156,6 +156,35 @@ func TestAsyncAgreement(t *testing.T) {
 	}
 }
 
+func TestAsyncAgreementXXX(t *testing.T) {
```

Comment:
I'll delete this before merging. It's a clone of the above, specialised to specific failing instances and with no parallelism so the logs are readable. FYI @masih

Reply:
Alternatively, use `F3_TEST_REPETITION_PARALLELISM=1` to serialise repetitions?